coralproject/talk
# [CORL-2077] Nonexistent ignoredUsers

## What does this PR do?

This PR addresses Jira ticket CORL-2077, in which users deleted via migration are present in existing users' `ignoredUsers` array.

## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## How do I test this PR?

1. As user A, ignore existing user B.
2. Delete user B (manually, not through the API)
3. As user A, view your profile (there should be no error, and the deleted user should no longer
